test/functional/wallet_basic.py

```diff
@@ -92,13 +93,13 @@ def run_test(self):
         assert_equal(txout['value'], 50)
 
         # Send 21 BTC from 0 to 2 using sendtoaddress call.
-        # Locked memory should use at least 32 bytes to sign each transaction
+        # Locked memory should increase to sign transactions
         self.log.info("test getmemoryinfo")
         memory_before = self.nodes[0].getmemoryinfo()
         self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 11)
         mempool_txid = self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 10)
         memory_after = self.nodes[0].getmemoryinfo()
-        assert(memory_before['locked']['used'] + 64 <= memory_after['locked']['used'])
+        assert_greater_than(memory_after['locked']['used'], memory_before['locked']['used'])
```

Comment:
Shouldn't the right argument be `+63`?

Reply:
The memory usage change between these two calls is actually 32.

I'm not entirely sure why `getmemoryinfo` is part of the `wallet_basic` test. How bitcoin core allocates locked memory pages seems purely an implementation detail and shouldn't cause functional wallet tests to fail.

To minimize the delta in this PR, I changed the equality check to a looser inequality check, but I think this test case should be moved out of `wallet_basic.py` entirely in a future PR.

Reply:
sounds like a good one for someone to do =).

Curious how the test passed before if it was only 32 bytes?

Reply:
The difference was 64 bytes before the change to the way the test generates blocks.

Reply:
makes sense.